airflow/models/dag.py

```diff
@@ -105,7 +105,8 @@
 
 
 DagStateChangeCallback = Callable[[Context], None]
-ScheduleInterval = Union[None, str, timedelta, relativedelta]
+MultiCron = Union[List[str], Set[str], Tuple[str, ...]]
```

Comment:
This additional type alias doesn’t seem to be necessary?

(Also this can probably be simply `Collection[str]`)